README.md

```diff
@@ -1,2 +1,31 @@
 # design-docs
+
 Workspace for all Storj design documents
+
+## Conventions
+
+- All the documents are in the root of this repository.
+- Documents are in [Github flavored markdown](https://github.github.com/gfm/).
+- The name of the documents starts with the date that the writer starts to write it, with the format
+  `YYYYMMDD-`, followed with the title of the document transformed to lowercase and hyphen-separated
+  (replace white spaces with `-`) and the `.md` file extension.
+- Each document has a frontmatter with the following required fields:
+  - `tags`: A list of tags which classifies the document.
+
+  The title of the document must be separated with one blank line of the frontmatter separator.
+- Additional files linked by the document are accepted, however, we discourage them when there are
+  valid alternatives for not requiring them. For example, [mermaid diagrams](https://mermaid.js.org/)
+  instead of diagrams that cannot be expressed in the same markdown document and not rendered by
+  Github.
+
+  The additional files must be store in a directory located in the root of this repository with the
+  same name of the document without the `.md` extension.
+
+
+### Old blueprints
+
+Storj used to write design documents, called blueprints, in a directory of the
+[storj/storj repository](https://github.com/storj/storj).
+
+All the documents that we ported into this repository are stored in the directory `old-blueprints`
```

Comment:
Do we need the separate folder? Maybe we can separate out "non-functional" designs i.e. things that have been changed?

Reply:
what do you suggest? bring them to the root folder without any change? bring them to the root folder changing their name, content, etc.?

Reply:
Bring them into root folder with the first commit date as the header. Any docs that are not applicable (e.g. code has been deleted), move to `archived/` folder.

Reply:
And also add tags.

Reply:
The format is going to be different, we are going to create a template for our current format in #5 and I will modify the README to refer to it.

If that's fine, I don't have any inconvenient, but I'll create a separate issue and delete this sentence.

Reply:
I think having a different format / template for old docs should be fine. Being able to easily find the relevant info is somewhat more important.

Reply:
Section deleted and issue created https://github.com/storj/design-docs/issues/8